### PR TITLE
Acceptance test modifications

### PR DESCRIPTION
**Pull request by rmacdon465**
Added Acceptance tests and modified helper

### CODE REVIEW

File: tests/_support/AcceptanceTester.php [hunk 1]
```diff
@@ -27,6 +27,8 @@ class AcceptanceTester extends \Codeception\Actor {
 		'allDay'  => false,
 	);
 
+
```
Comment (barryhughes) [inline tests/_support/AcceptanceTester.php]: Whitespace - do we need the extra lines here?

File: tests/_support/AcceptanceTester.php [hunk 2]
```diff
@@ -68,6 +71,43 @@ public function generateEvent() {
 		return array_merge( self::$defaultEvent, $random );
 	}
 
+	/**
+	 * Update Event
+	 */
+	public function edit_event( $updateEvent )  {
+		$originalTitle		= $updateEvent['originalTitle'];
```
Comment (barryhughes) [inline tests/_support/AcceptanceTester.php]: Whitespace - mix of tabs and spaces? A few things seem misaligned.

File: tests/_support/AcceptanceTester.php [hunk 3]
```diff
@@ -136,12 +176,87 @@ public function upload_csv( $type, $file ) {
...
+		if ( is_null( $venue['venueTitle'] ) ) {
+			throw new \InvalidArgumentException('Missing Arguments!  Need a title at least');
+		} 
+
+		$I = $this;
+		$I->amOnPage( '/wp-admin/post-new.php?post_type=tribe_venue' );
+		$I->fillField( 'post_title', $venue['venueTitle'] );
+		$I->fillField( 'venue[Address]', $venue['venueAddress'] );
+		$I->fillField( 'venue[City]', $venue['venueCity'] );
+		// TO DO - Add selector for "Parent"
```
Comment (barryhughes) [inline tests/_support/AcceptanceTester.php]: Switch to the `@todo` convention?

File: tests/_support/AcceptanceTester.php [hunk 4]
```diff
@@ -68,6 +71,43 @@ public function generateEvent() {
...
+
+		$I = $this;
+
+		$I->amOnPage( '/wp-admin/edit.php?post_type=tribe_events' );
+		$I->fillField( 's', $originalTitle );
+		$I->click( '#search-submit' );
+		$I->click( $originalTitle );
+
+		//We should now be on the edit page but check
+		//$I->see( $updateEvent['originalTitle'] );
```
Comment (barryhughes) [inline tests/_support/AcceptanceTester.php]: This and the other `$I->see()` on line 102 are commented out - is there a reason for that/do they currently fail and we're going to address later?

File: tests/_support/AcceptanceTester.php [hunk 5]
```diff
@@ -136,12 +176,87 @@ public function upload_csv( $type, $file ) {
...
+			throw new \InvalidArgumentException('Missing Arguments!  Need a name at least');
+		} 
+
+		$I = $this;
+		$I->amOnPage( '/wp-admin/edit-tags.php?taxonomy=post_tag&post_type=tribe_events' );
+		$I->fillField( 'tag-name', $tag['tagName'] );
+		$I->fillField( 'slug', $tag['tagSlug'] );
+		$I->fillField( 'description', $tag['tagDescription'] );
+	
+
```
Comment (kgneil) [inline tests/_support/AcceptanceTester.php]: whitespace

File: tests/_support/AcceptanceTester.php [hunk 6]
```diff
@@ -136,12 +176,87 @@ public function upload_csv( $type, $file ) {
...
+		$I->fillField( 'tag-name', $tag['tagName'] );
+		$I->fillField( 'slug', $tag['tagSlug'] );
+		$I->fillField( 'description', $tag['tagDescription'] );
+	
+		
+		$I->click( '#submit' );
+		$I->see( $tag['tagName'] );
+		//$I->see( $tag['tagSlug'] );
+		//$I->see( $tag['tagDescription'] );
+
```
Comment (kgneil) [inline tests/_support/AcceptanceTester.php]: Whitespace

File: tests/_support/AcceptanceTester.php [hunk 7]
```diff
@@ -68,6 +71,43 @@ public function generateEvent() {
...
+	/**
+	 * Update Event
+	 */
+	public function edit_event( $updateEvent )  {
+		$originalTitle		= $updateEvent['originalTitle'];
+		$newTitle			= $updateEvent['newTitle'];
+		$content			= $updateEvent['content'];
+		$allDay 			= $updateEvent['allDay'];
+		if(empty($updateEventp['originalTitle']) || empty($updateEvent['newTitle'])){
+            throw new \InvalidArgumentException('Title missing');
```
Comment (kgneil) [inline tests/_support/AcceptanceTester.php]: Not to sure if it is just my view but indentation is off here

File: tests/_support/AcceptanceTester.php [hunk 8]
```diff
@@ -68,6 +71,43 @@ public function generateEvent() {
...
+		$I->amOnPage( '/wp-admin/edit.php?post_type=tribe_events' );
+		$I->fillField( 's', $originalTitle );
+		$I->click( '#search-submit' );
+		$I->click( $originalTitle );
+
+		//We should now be on the edit page but check
+		//$I->see( $updateEvent['originalTitle'] );
+
+		$I->fillField( 'post_title', $updateEvent['newTitle'] );
+		$I->fillField('content', $updateEvent['content'] ); // need to target WYSIWYG instance
```
Comment (kgneil) [inline tests/_support/AcceptanceTester.php]: Just need a space after the first (   
$I->fillField( 'content', $updateEvent['content'] );

File: tests/_support/AcceptanceTester.php (hunk 1 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: I believe I have taken care of this.

File: tests/_support/AcceptanceTester.php (hunk 2 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: I believe I have taken care of this.

File: tests/_support/AcceptanceTester.php (hunk 3 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: I believe I have taken care of this.

File: tests/_support/AcceptanceTester.php (hunk 4 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: They are currently failing but I have it on my list to figure out why.  However, it's not super high on the list.

File: tests/_support/AcceptanceTester.php (hunk 7 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: I believe I have taken care of this.

File: tests/_support/AcceptanceTester.php (hunk 8 above)
Comment (rmacdon465) [inline tests/_support/AcceptanceTester.php]: I believe I have taken care of this.

**Comment by rmacdon465** [comment]
Killing this PR due to new branch.  Opening new in a few

**Pull request closed without merging**